include/dosbox.h

```diff
@@ -63,37 +63,34 @@ int64_t DOSBOX_GetTicksDone();
 void DOSBOX_SetTicksDone(const int64_t ticks_done);
 void DOSBOX_SetTicksScheduled(const int64_t ticks_scheduled);
 
-enum SVGACards {
-	SVGA_None,
-	SVGA_S3Trio,
-	SVGA_TsengET4K,
-	SVGA_TsengET3K,
-	SVGA_ParadisePVGA1A
-}; 
-
-extern SVGACards svgaCard;
-extern bool mono_cga;
-
-enum MachineType {
+enum class MachineType {
 	// In rough age-order: Hercules is the oldest and VGA is the newest
 	// (Tandy started out as a clone of the PCjr, so PCjr came first)
-	MCH_INVALID = 0,
-	MCH_HERC    = 1 << 0,
-	MCH_CGA     = 1 << 1,
-	MCH_TANDY   = 1 << 2,
-	MCH_PCJR    = 1 << 3,
-	MCH_EGA     = 1 << 4,
-	MCH_VGA     = 1 << 5,
+	Invalid,
+	Hercules,
+	Cga,
+	Tandy,
+	PcJr,
```

Comment:
I'd just go `Pcjr`. It's stylized as `PCjr` originally 😄 I'm using this form already in `GraphicsStandard::Pcjr`.